src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,24 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("The user in null");
+        }
+        if (user.getPassword().length() < 6) {
+            throw new RuntimeException("The password is too short");
+        }
+        if (user.getAge() < 18) {
+            throw new RuntimeException("The user is too young");
+        }
+        if (storageDao.get(user.getLogin()) != null) {
+            throw new RuntimeException("Such user is already exist");
+        }
```

Comment:
This is ok, but you can split this code into simple methods, so you avoid complicated and repeated if statements.